From 5379a62709d93c87a1354c5f0acb9d1e64d76df6 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Mon, 18 Dec 2023 21:06:57 +0000
Subject: [PATCH 1/1] tsp-page: Replace gnome-fs-share with folder-publicshare
 icon

gnome-fs-share was removed with nautilus 3.1 many years ago, as such
icon themes have begun to stop providing this icon.

Replace with a modern suitable alternative.
---
 thunar-plugin/tsp-page.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thunar-plugin/tsp-page.c b/thunar-plugin/tsp-page.c
index 33c397b..afd3b02 100644
--- a/thunar-plugin/tsp-page.c
+++ b/thunar-plugin/tsp-page.c
@@ -143,7 +143,7 @@ tsp_page_init (TspPage *page)
   gtk_box_pack_start (GTK_BOX (vbox1), hbox1, FALSE, TRUE, 0);
 
   widget = gtk_image_new ();
-  gtk_image_set_from_icon_name (GTK_IMAGE (widget), "gnome-fs-share", GTK_ICON_SIZE_DIALOG);
+  gtk_image_set_from_icon_name (GTK_IMAGE (widget), "folder-publicshare", GTK_ICON_SIZE_DIALOG);
   gtk_widget_set_halign (widget, GTK_ALIGN_START);
   gtk_box_pack_start (GTK_BOX (hbox1), widget, FALSE, FALSE, 0);
 
-- 
2.43.0

